### PR TITLE
CYF-LDN7-Bisi Olajide-Update homework.md-WEEK 2

### PR DESCRIPTION
**Pull request by bisioolajide**
Homework corrections

# Your Details

- Your Name: Bisi Olajide
- Your City: London
- Your Slack Name: bisioolajide

# Homework Details

- Module: GIT
- Week: 1

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,77 +4,78 @@
...
+Cascading Style Sheets (CSS) is used to define styles for your web pages, including the design, layout and variations in display for different devices and screen sizes
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+.h1 {
```
Comment (NeginDn) [inline homework.md:38]: I think you have typo here

File: homework.md [hunk 2]
```diff
@@ -86,19 +87,19 @@ You send the changes of your local repository to Github
 
 What is Github?
 
-A place for us to store and share our code.
+GitHub is one website where you can use Git.
```
Comment (NeginDn) [inline homework.md:90]: I suggest to double check this answer and make it a bit more understanable/detailed

File: homework.md [hunk 3]
```diff
@@ -86,19 +87,19 @@ You send the changes of your local repository to Github
...
 
-Send you code to Github
+Save your changes to the local version of the repository
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio
```
Comment (NeginDn) [inline homework.md:102]: The full name is Visual Studio Code

**Review by NeginDn — COMMENTED** [review]
Hi Bisi,
Great job with the homework, I just made a few suggestion in case you want to use it :)

**Comment by bisioolajide** [comment]
Thank's Negin, yes will go back and update all your suggestions.

Regards,
Bisi

**Pull request closed without merging**